### ISSUE TITLE
learning_rate has changed

### ISSUE THREAD

**Issue by summerstay** [dribnet/pixray#17]
I was using a colab notebook where I set the learning_rate. But some time in the last month or so the learning rate I had before is now much too small. How has learning_rate changed?
